utils/build/docker/nodejs/parametric/server.js

```diff
@@ -285,6 +285,25 @@ app.post('/trace/otel/set_attributes', (req, res) => {
   res.json({});
 });
 
+app.get('/trace/config', (req, res) => {
+  const dummyTracer = require('dd-trace').init()
+  const config = dummyTracer._tracer._config
+  res.json( { 
+    'dd_service': config ? config.service : null,
```

Comment:
Is this valid JavaScript? If so, it may be cleaner

```suggestion
    'dd_service': config?.service ?? null,
```